client/src/components/Page/RunSettingsPage.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+// import styled from "styled-components/macro";
+
+import AppHeader from "../../components/AppHeader/AppHeader";
+import BottomNav from "../../components/BottomNav/BottomNav";
+import Button from "../../components/Button/Button";
+import Input from "../../components/Input/Input";
+
+export default function RunSettingsPage() {
+  return (
+    <div>
+      <AppHeader></AppHeader>
+      <Input></Input>
+      {/* MAP */}
+      <Input></Input>
```

Comment:
`input` is a void element. 
The other elements are not visible, because you have no content/children.

```suggestion
      <Input />
```